### ISSUE TITLE
Video control stopped on www.crunchyroll.com

### ISSUE THREAD

**Issue by duttonw** [igrigorik/videospeed#782]
With the recent malware in extensions cropping up.

I've elected to have most of my extensions be enabled on click or giving a safe list. 

I recently visited https://www.crunchyroll.com/ and was trying to speed up the videos on it, sadly it seems the plugin in google chrome did not interact with the html5 video player that crunchyroll uses. It was working a week or two before. 

Does anyone else have any issues with this site also?

**Comment by Tittoes** [comment]
yes it also doesnt work on crunchyroll for me hope it gets fixe.  im pretty sure they are still using html5? im not sure

**Comment by duttonw** [comment]
@Tittoes I found a solution for crunchyroll (CR) , with safelist (or click) option enabled, you also need to enabled the "Allow access to file URLs" via chrome://extensions/?id=nffaoalbilbmmfgbnbgppjihopabppdk 

I found this out after whitelisting all domains that they loaded their videos from. If you opened it directly it would pop up (But video would not play). CR is doing in memory file processing of the video stream outside of html5 spec (like prime video or netflix) for drm/security of the video stream. 

I believe they are doing what you can see on https://developers.google.com/web/updates/2011/08/Saving-generated-files-on-the-client-side where you can create a file in memory for x seconds, and add it to the dom playlist which the player seamlessly loads without stuttering etc.
